app/src/components/market/sections/market_navigation/index.tsx

```diff
@@ -21,15 +23,18 @@ const MarketTab = styled.div<{ active: boolean }>`
 
 interface Props {
   activeTab: string
-  marketAddress: string
   hasWinningOutcomes?: Maybe<boolean>
-  isQuestionFinalized: boolean
-  resolutionDate: Date
+  marketMakerData: MarketMakerData
   switchMarketTab: (arg0: string) => void
 }
 
 export const MarketNavigation = (props: Props) => {
-  const { activeTab, hasWinningOutcomes, isQuestionFinalized, resolutionDate, switchMarketTab } = props
+  const { activeTab, hasWinningOutcomes, marketMakerData, switchMarketTab } = props
+  const { isQuestionFinalized, question } = marketMakerData
+  const currentTimestamp = new Date().getTime()
+
+  const isOpen = question.resolution.getTime() > currentTimestamp
```

Comment:
It's not correct to check if it's opened market using resolution.getTime()

Please check openingTimestamp
https://github.com/protofire/omen-exchange/blob/bc7aa2babe31353b587e7c2b69523489ea26a54c/app/src/queries/markets_home.ts#L87

Reply:
![Screenshot 2020-11-17 at 13 08 28](https://user-images.githubusercontent.com/33226956/99388503-fcd69b00-28d5-11eb-9cff-5c432d521a64.png)
should it then also be changed here? In market top details open. @KadenZipfel @liaojikunwork

Reply:
`question.resolution` should actually just be `openingTimestamp` converted to a `Date` object